data/modules/Debug/DebugCommodityPrices.lua

```diff
@@ -260,13 +248,11 @@ local function show_system_info(min, sys, str)
 	ui.text("Distance:\t " .. dist .. " ly")
 	ui.text("Faction:\t " .. sys.faction.name)
 	ui.text("Government:\t " .. string.lower(sys.govtype):gsub("^%l", string.upper)) -- PolitGovType
-	ui.text("") 	-- PolitEcon
 
 	-- Add from C++ side, E.g:
 	-- System type: Mining colony
 	-- Government type: Earth Federation Democrac
 	-- Economy type: capitalist
-	ui.text("")
 end
 
 local function show_details_on_commodity(commodities, clicked)
```

Comment:
Nitpick: if I understand this code correctly, `clicked` should probably be something like `selected`. Typically we use "clicked" for an action happening to a specific thing (was-clicked) rather than a noun.